apple/scripts/ios_package_remote.action.sh

```diff
@@ -15,6 +15,26 @@
 # limitations under the License.
 
 # Packages the Outline iOS client on Travis. Expects to be invoked through `npm run action`.
-./apple/scripts/install_fastlane.sh -p ios
+PLATFORM=
```

Comment:
forgive me, but this block feels like something we could DRY up? i'm not exactly sure how to go about that in bash, however. maybe we save that for a later PR

Reply:
Remove. This is an ios release script.

Reply:
One way would be to use env vars, but that is really hard to track.
I think a lot can be deleted. And perhaps the webpack script could handle our mode strings somehow. Or we could have a script that takes a build mode string and converts to a webpack mode string

Reply:
I'm thinking that in the future we could use something like dotenv to merge envs from different variant dimensions to create one env for the entire build. One of such variables would be the webpack mode.

Reply:
Currently I'm using this script in the ios CI build - i'll change that to gulp but it might take a second to figure that out. it's possible junyi's recent changes fixed gulp in the CI

Reply:
Is there anything stopping us from moving all of these actions to node?

Reply:
Currently I'm using this script in the ios CI build - i'll change that to gulp but it might take a second to figure that out. it's possible recent changes fixed gulp in the CI